integration-cli/utils.go

```diff
@@ -83,3 +84,26 @@ func parseCgroupPaths(procCgroupData string) map[string]string {
 func runAtDifferentDate(date time.Time, block func()) {
 	integration.RunAtDifferentDate(date, block)
 }
+
+func findCgroupMountpoints() map[string]string {
+	cgMounts, err := cgroups.GetCgroupMounts()
```

Comment:
The utility probably need to be split up to utils, utils_windows and utils_linux, however the problem remains that the test should depend on the *daemon* implementing it, and the test is run on the client, so that makes it tricky (probably the only solution would be to expose this information through the API, but that may be too much it its only used for integration tests